clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -1760,24 +1767,38 @@ def validate_args(argv, command, expected_nr):
         raise Exception("p4rt-ctl: '{}' command requires at least {} "
                         "arguments".format(command, expected_nr))
 
+# Global variable
+grpc_server_addr = ""
 
 def main():
+    global grpc_server_addr
     if len(sys.argv) < 2:
         print("p4rt-ctl: missing command name; use --help for help")
         sys.exit(1)
     parser = argparse.ArgumentParser(usage=USAGE)
-    parser.add_argument('command', help='Subcommand to run')
-
-    args = parser.parse_args(sys.argv[1:2])
-    if args.command not in all_commands.keys():
+    parser.add_argument('-g', '--grpc_addr', required=False, type=str, help="P4Runtime gRPC server address, format : <server IP>:<port>")
+    parser.add_argument('command', nargs='*', help='Subcommand to run')
+    args = parser.parse_args()
+    if args.grpc_addr is None:
```

Comment:
Please add a comment regarding this change.

Reply:
What kind of comment you are looking for? Commit message explains new parameter and help will also show what's expected

Reply:
Comment on what is this change about and how user should use it?
Similar comment available on "p4ctl_add_entry" API on how to use this parameter.

We cant go back and view commit message on the usage right?
As we dont have any specific documentation for p4rt-ctl, how will the user know how to use this command?

If not comment, maybe adding it in helper command, as part of USAGE options?
"def usage"/"usage: p4rt-ctl [OPTIONS] COMMAND [ARG...]"